README.md

```diff
@@ -57,31 +69,49 @@ For further usage examples for Scala MaxMind IP Lookups, please see the tests in
 The signature is as follows:
 
 ```scala
-class IpLookups(geoFile: Option(File), ispFile: Option(File), 
-                orgFile: Option(File), domainFile: Option(File), 
-                memCache: Boolean = true, lruCache: Int = 10000)
+class IpLookups(
+  geoFile: Option(File), 
+  ispFile: Option(File), 
+  orgFile: Option(File), 
+  domainFile: Option(File), 
```

Comment:
can we fix those into `Option[File]`

Reply:
The README needs to be generally worked over. I'm not going to take that on.